recipes/libvips/meta.yaml

```diff
@@ -0,0 +1,121 @@
+{% set name = "libvips" %}
+{% set version = "8.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libvips/libvips/releases/download/v{{ version }}/vips-{{ version }}.tar.gz
+  sha256: a0ee255a2a1ebfea5b2dff2a780824d7157a78c010d7ddd531279aacefbf2539 
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bzip2 
+    - cairo 
+    - cfitsio 
+    - expat
+    - fftw
+    - fontconfig
+    - freetype
+    - gdk-pixbuf
+    - giflib
+    - glib
+    - hdf5
+    - imagemagick
+    - jpeg
+    - libmatio
+    - libpng
+    - librsvg 
+    - libtiff
+    - libwebp
+    - make
+    - pkg-config
+    - pango =1.40.14
+    - poppler
+    - zlib
+  host:
+    - bzip2 
+    - cairo 
+    - cfitsio 
+    - expat
+    - fftw
+    - fontconfig
+    - freetype
+    - gdk-pixbuf
+    - giflib
+    - glib
+    - hdf5
+    - imagemagick
+    - jpeg
+    - libmatio
+    - libpng
+    - librsvg 
+    - libtiff
+    - libwebp
+    - pango =1.40.14
+    - poppler
+    - zlib
+  run:
+    - bzip2 
+    - cairo 
+    - cfitsio 
+    - expat
+    - fftw
+    - fontconfig
+    - freetype
+    - gdk-pixbuf
+    - giflib
+    - glib
+    - hdf5
+    - imagemagick
+    - jpeg
+    - libmatio
+    - libpng
+    - librsvg 
+    - libtiff
+    - libwebp
+    - pango =1.40.14
+    - poppler
+    - zlib
+
+test:
+  commands:
+    - vips --version
+
+about:
+  home: https://libvips.github.io/libvips
+  license: LGPL-2.1
+  license_family: LGPL
+  license_file: COPYING
```

Comment:
Basically, this line will look at the top level of the source, and find that the COPYING file exists.